src/models/meme.ts

```diff
@@ -0,0 +1,20 @@
+class MemeResource {
+  id: string;
+  title: string;
+  description: string;
+  link: string;
+  nsfw: boolean;
+  type: string;
+  created_at: Date;
```

Comment:
Not needed in constructor because of `table.dateTime("created_at").defaultTo(knex.fn.now());` in `create-meme-table.ts`?

Reply:
Huh, I missed this. I would also add it in the constructor. Maybe as a default value 🤔

Reply:
Done. The object default value for `created_at` is now `new Date()`. The value is represented as passed milliseconds(!) since 01.01.1970 00:00:00.